update/update.sh

```diff
@@ -1,9 +1,17 @@
-#!/usr/bin/env bash
-set -euo pipefail
+#!/bin/sh
+set -eu
```

Comment:
Give the Docker image is supplied to the plugin, there's no guarantee that Bash will be installed. Let's drop back to `bin/sh` and make sure this script is portable.